python3.qcelemental fails to build

Changes on nixpkgs master (somewhere between f5f6dc053b1a0eca03c853dad710f3de070df24e and c5147860e23ed75ce9d40298c66b416c00be1167) broke the `qcelemental` package (and subsequently psi4). See tail of the build log below.

CC @sheepforce 

```
________ ERROR collecting qcelemental/tests/test_molparse_to_string.py _________
qcelemental/tests/test_molparse_to_string.py:270: in <module>
    "subject1": qcel.models.Molecule(
qcelemental/models/molecule.py:294: in __init__
    super().__init__(**kwargs)
/nix/store/69gpcv5qv634nfib8prwflyn7m88v33m-python3.8-pydantic-1.8/lib/python3.8/site-packages/pydantic/main.py:391: in __init__
    raise validation_error
E   pydantic.error_wrappers.ValidationError: 2 validation errors for Molecule
E   connectivity -> 0
E     '<' not supported between instances of 'tuple' and 'int' (type=type_error)
E   connectivity -> 1
E     '<' not supported between instances of 'tuple' and 'int' (type=type_error)
=============================== warnings summary ===============================
qcelemental/molparse/from_arrays.py:599: 20 warnings
  /build/qcelemental-0.17.0/qcelemental/molparse/from_arrays.py:599: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    npgeom = np.array(geom, copy=copy, dtype=np.float).reshape((-1, 3))

qcelemental/molparse/from_arrays.py:698: 20 warnings
  /build/qcelemental-0.17.0/qcelemental/molparse/from_arrays.py:698: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    "elea": np.array(A, dtype=np.int),

qcelemental/molparse/from_arrays.py:699: 20 warnings
  /build/qcelemental-0.17.0/qcelemental/molparse/from_arrays.py:699: DeprecationWarning: `np.int` is a deprecated alias for the builtin `int`. To silence this warning, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    "elez": np.array(Z, dtype=np.int),

qcelemental/molparse/from_arrays.py:701: 20 warnings
  /build/qcelemental-0.17.0/qcelemental/molparse/from_arrays.py:701: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    "mass": np.array(mass, dtype=np.float),

qcelemental/molparse/from_arrays.py:702: 20 warnings
  /build/qcelemental-0.17.0/qcelemental/molparse/from_arrays.py:702: DeprecationWarning: `np.bool` is a deprecated alias for the builtin `bool`. To silence this warning, use `bool` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.bool_` here.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    "real": np.array(real, dtype=np.bool),

qcelemental/tests/test_datum.py:57
  /build/qcelemental-0.17.0/qcelemental/tests/test_datum.py:57: DeprecationWarning: `np.float` is a deprecated alias for the builtin `float`. To silence this warning, use `float` by itself. Doing this will not modify any behavior and is safe. If you specifically wanted the numpy scalar type, use `np.float64` here.
  Deprecated in NumPy 1.20; for more details and guidance: https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations
    (("ndarray", "1/m"), np.arange(4, dtype=np.float) * 400 / 3),

-- Docs: https://docs.pytest.org/en/stable/warnings.html
=========================== short test summary info ============================
ERROR qcelemental/tests/test_molparse_to_string.py - pydantic.error_wrappers....
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
======================== 101 warnings, 1 error in 1.45s ========================
builder for '/nix/store/r0c6xz0zz2lkw1x9m2d670qib039a3q8-python3.8-qcelemental-0.17.0.drv' failed with exit code 2
```

--- a/qcelemental/default.nix
+++ b/qcelemental/default.nix
@@ -12,7 +12,7 @@
 }:
 buildPythonPackage rec {
   pname = "qcelemental";
-  version = "0.17.0";
+  version = "0.19.0";
 
   checkInputs = [
     pytestrunner
@@ -30,7 +30,7 @@ buildPythonPackage rec {
 
   src = fetchPypi  {
     inherit pname version;
-    sha256 = "0xsxmj1l6l72ga06a1gy2kf7lqdgnx8hcgqw0ikvzgs0xj2w4421";
+    sha256 = "1ljxwhiz1689qijjqzmzydn8q963xx7zxizjnjq0vqg5py9pkyc5";
   };
 
   doCheck = true;

